src/package/cppwinrt/nuget/Microsoft.Windows.CppWinRT.targets

```diff
@@ -315,7 +310,7 @@ $(XamlMetaDataProviderPch)
     <!--Insert Midl /references to Platform WinMDs, library reference WinMDs, and direct reference WinMDs-->
     <Target Name="CppWinRTSetMidlReferences"
             BeforeTargets="Midl"
-            DependsOnTargets="GetCppWinRTPlatformWinMDReferences;CppWinRTAddXamlMetaDataProviderIdl;GetCppWinRTProjectWinMDReferences;GetCppWinRTDirectWinMDReferences;$(CppWinRTSet_MidlReferencesDependsOn)">
+            DependsOnTargets="GetCppWinRTPlatformWinMDReferences;GetCppWinRTDirectWinMDReferences;GetCppWinRTProjectWinMDReferences;CppWinRTAddXamlMetaDataProviderIdl;$(CppWinRTSetMidlReferencesDependsOn)">
```

Comment:
I don't see any references to CppWinRTSet_MidlReferencesDependsOn, so probably safe to rename, but we should be very intentional about that.

Reply:
Ah good catch. I can add the underscore back. Didn't realize it had an underscore there before. If there are no refs let's keep the rename.